app/lib/twiddle-resolver.js

```diff
@@ -6,50 +6,44 @@ export default Resolver.extend({
   },
 
   resolveOther (parsedName) {
-    if (parsedName.fullName === 'router:main') {
```

Comment:
`twiddle-resolver` doesn't do anything anymore as the application is now using the normal resolver